flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java

```diff
@@ -825,7 +811,7 @@ private void snapshotTaskState(
             long checkpointID,
             CheckpointStorageLocation checkpointStorageLocation,
             CheckpointProperties props,
-            Execution[] executions,
+            List<Execution> executions,
```

Comment:
Here, a renaming might be helpful. This should really be `tasksToTrigger`, right?